[docs] update the Android build guide to set up NDK (#244)
openthread/ot-commissioner: android/BUILDING.md

- macOS or Linux computer.
- Install latest Android Studio from [here](https://developer.android.com/studio).

> - Install Android NDK by following https://developer.android.com/studio/projects/install-ndk#default-version

Reviewer: ```suggestion
- Install Android NDK by following https://developer.android.com/studio/projects/install-ndk#default-version.
```

ANDROID_ABI=arm64-v8a ANDROID_NDK_HOME=$HOME/Library/Android/sdk/ndk-bundle ./build-commissioner-libs.sh
```

> Note:
>

> > 1.  you need to set env `ANDROID_ABI` to the ABI of your phone. This value can be got with command `adb shell getprop ro.product.cpu.abi` after connecting the phone to your computer.

Reviewer: ```suggestion
> 1.  You need to set `ANDROID_ABI` to the ABI of your phone. This value can be retrieved using the command `adb shell getprop ro.product.cpu.abi` after connecting the phone to your computer.
```

> > 2.  you need to set `ANDROID_NDK_HOME` to your Android NDK directory. This is typically `$HOME/Android/sdk/ndk/xx.x.xxxxx` (where `xx.x.xxxxx` is the NDK version) on Linux and Mac OS or `$HOME/Library/Android/sdk/ndk-bundle` on older OS versions.

Reviewer: ```suggestion
> 2.  You need to set `ANDROID_NDK_HOME` to your Android NDK directory. This is typically `$HOME/Android/sdk/ndk/xx.x.xxxxx` (where `xx.x.xxxxx` is the NDK version) on Linux and Mac OS or `$HOME/Library/Android/sdk/ndk-bundle` on older OS versions.
```
